instrumentation/jsonrpc4j-1.6/testing/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+plugins {
+  id("otel.java-conventions")
+}
+
+val jsonrpcVersion = "1.6"
+
+dependencies {
+  api(project(":testing-common"))
+
+  implementation("com.github.briandilley.jsonrpc4j:jsonrpc4j:$jsonrpcVersion")
+  implementation("com.fasterxml.jackson.core:jackson-databind:2.13.3")
+
+  // ...
```

Comment:
Is it unnecessary?

Reply:
ah yes, you're right.I didn't notice that actually `import io.opentelemetry.testing.internal.jackson.databind.JsonNode` not the one from jackson.

Reply:
seems it's still necessary. If I delete this line, there would be a compile error:
```
/opentelemetry-java-instrumentation/instrumentation/jsonrpc4j-1.6/testing/src/main/java/io/opentelemetry/instrumentation/jsonrpc4j/v1_6/AbstractJsonRpcTest.java:41: error: cannot access ObjectMapper
          configureServer(new JsonRpcBasicServer(calculator, CalculatorService.class));
                          ^
    class file for com.fasterxml.jackson.databind.ObjectMapper not found
```

Reply:
What I say here is line 13.